File: checker_test/ac/submission.java
import java.util.Scanner;
import java.util.ArrayList;

public class submission {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        ArrayList<Integer> arr = new ArrayList<>();

        for (int i = 0; i < n; i++) {
            int x = sc.nextInt();
            arr.add(x);
        }

        sc.close();

        int maxx = arr.get(0);
        int maxIndex = 0;
        for (int i = 1; i < n; i++) {
            if (arr.get(i) > maxx) {
                maxx = arr.get(i);
                maxIndex = i;
            }
        }

        System.out.println(maxIndex);
    }
}

File: checker_test/ac/submission.lua
n = io.read("*number")

arr = {}

for i = 1, n do
    arr[i] = io.read("*number")
end

max = arr[1]
max_index = 1

for i = 2, n do
    if arr[i] > max then
        max = arr[i]
        max_index = i
    end
end

io.write(max_index)